pyat/at/physics/fastring.py

```diff
@@ -100,3 +101,136 @@ def fast_ring(ring: Lattice, split_inds=[]) -> Tuple[Lattice, Lattice]:
                          split_inds=split_inds,
                          detuning_elem=detuning_elem)
     return fastringnorad, fastringrad
+
+
+def simple_ring(energy: float, circumference: float, harmonic_number: int,
+                Qx: float, Qy: float, Vrf: float, alpha: float,
```

Comment:
Vrf and harmonic_number can be scalars or Sequence of scalars

Reply:
done